terraform/critical_prod/main.tf

```diff
@@ -21,7 +21,8 @@ module "critical" {
     "arn:aws:iam::653428163053:user/echo-fs",
   ]
 
-  replica_glacier_read_principals = []
+  replicator_primary_task_role_arn = "arn:aws:iam::975596993436:role/storage-prod-bag-replicator_primary_task_role"
```

Comment:
Where are these created? Could we pull them from that state?

Reply:
We could, but you’d create circular references between the two state files.

Stack depends on critical (to get the bucket/database) names, but then critical would depend on stack (to get these names).

Reply:
Maybe a note?